tests/pom.xml

```diff
@@ -102,37 +114,78 @@
+        <hawkular.port>8080</hawkular.port>
         <hawkular.path>/hawkular/btm/</hawkular.path>
         <hawkular.base-uri>http://${hawkular.host}:${hawkular.port}${hawkular.path}</hawkular.base-uri>
         <!-- IMPORTANT: The port must be the port offset + 8080. -->
         <wildfly.port.offset>1897</wildfly.port.offset>
         <!-- IMPORTANT: The management port must be the port offset + 9990. -->
         <wildfly.management.port>11887</wildfly.management.port>
+        <hawkular.home>${project.build.directory}/hawkular</hawkular.home>
+        <hawkular.wildfly>${hawkular.home}/wildfly-${version.org.wildfly}</hawkular.wildfly>
+        <hawkular.configuration>${hawkular.wildfly}/standalone/configuration</hawkular.configuration>
+        <hawkular.data>${hawkular.wildfly}/standalone/data</hawkular.data>
+        <hawkular.deployments>${hawkular.wildfly}/modules/org/hawkular/nest/main/deployments</hawkular.deployments>
       </properties>
       <build>
         <plugins>
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-resources-plugin</artifactId>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>unpack-hawkular</id>
+                <phase>pre-integration-test</phase>
+                <goals>
+                  <goal>unpack-dependencies</goal>
+                </goals>
+                <configuration>
+                  <includeGroupIds>org.hawkular.btm</includeGroupIds>
+                  <includeArtifactIds>hawkular-btm-dist</includeArtifactIds>
+                  <outputDirectory>
+                    ${hawkular.home}
+                  </outputDirectory>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-patch-plugin</artifactId>
+            <version>${plugin.version.org.apache.maven.plugins}</version>
             <executions>
               <execution>
-                <id>copy-data-dir</id>
+                <id>add-demo-user-realm</id>
                 <phase>pre-integration-test</phase>
                 <goals>
-                  <goal>copy-resources</goal>
+                  <goal>apply</goal>
                 </goals>
                 <configuration>
-                  <outputDirectory>${project.build.directory}/wildfly-data</outputDirectory>
-                  <overwrite>true</overwrite>
-                  <resources>
-                    <resource>
-                      <directory>${project.basedir}/src/test/wildfly-data</directory>
-                      <includes>
-                        <include>**/*</include>
-                      </includes>
-                    </resource>
-                  </resources>
+                  <patchFile>src/test/resources/hawkular-realm-patch.json</patchFile>
```

Comment:
So, I just checked it: the accounts version of this file already has jdoe:password (as the accounts distribution is meant for dev purposes only). For the main hawkular distribution, it's already shipping on master: https://github.com/hawkular/hawkular/blob/master/dist/assembly.xml#L184